packages/Datadog.Unity/Editor/DatadogHelpStrings.cs

```diff
@@ -0,0 +1,93 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-Present Datadog, Inc.
+
+// ReSharper disable InconsistentNaming
+namespace Datadog.Unity.Editor
+{
+    /// <summary>
+    /// Class to make editing tooltips and help strings easier.
+    /// </summary>
+    public static class DatadogHelpStrings
+    {
+        public const string EnabledTooltip =
+            "Enable or disable the Datadog Unity SDK. When disabled, the SDK will not send any data to Datadog.";
+
+        public const string OutputSymbolsTooltip =
+            "Whether to output symbol files as part of the build. Uploading these files to Datadog enables symbolication "
+            + "and file/line mapping features in Datadog Error Tracking.";
+
+        public const string ClientTokenTooltip = "This is your API key from Datadog.";
```

Comment:
Should we state API Key here? I'm wondering if customer will actually get confused and use an API key instead of their client token.

Reply:
+1 on this